doc/sphinx/src/interface/metadata.rst

```diff
@@ -55,6 +55,9 @@ mutually exclusive:
    cell-corners. The variable might be volume-averaged, or defined
    pointwise.
 
+For any type of topology, the flag ``Metadata::Fine`` can be set which 
+causes the field to have twice the resolution of normal fields.
+
```

Comment:
It'd be great to document what's currently tested/supported and what's "handle with care and expect sth go (silently) wrong"

Reply:
docs updated with user beware language.